examples/gno.land/p/demo/int256/arithmetic_test.gno

```diff
@@ -0,0 +1,215 @@
+package int256
+
+import "testing"
+
+type binOp2Test struct {
+	x, y, want string
+}
+
```

Comment:
This should not be a package-level global, please move it inside each individual test

Reply:
925889d084ce6684d445a933ac0890958838e21a

Reply:
b2156168aab867b305655cd16feec0c84de69d9b